extension.js

```diff
@@ -331,6 +364,20 @@ function unmaximize_window(win) {
         win.unmaximize(Meta.MaximizeFlags.VERTICAL);
 }
 
+function unmaximize_window_horizontally(win) {
+    if (!win || win !== current_window)
+        return;
+
+    if (!win.maximized_vertically)
```

Comment:
I think it should be `maximized_horizontally` here

Reply:
What a stupid mistake! >.<
Fixed.

Reply:
Yes. Very Stupid mistake. Fixed.